perun-core/src/main/java/cz/metacentrum/perun/core/impl/ConsentsManagerImpl.java

```diff
@@ -178,4 +181,50 @@ public ConsentHub createConsentHub(PerunSession sess, ConsentHub consentHub) {
 	public void checkConsentHubExists(PerunSession sess, ConsentHub consentHub) throws ConsentHubNotExistsException {
 		if(!consentHubExists(sess, consentHub)) throw new ConsentHubNotExistsException("ConsentHub not exists: " + consentHub);
 	}
+
+	@Override
+	public void addFacility(PerunSession sess, ConsentHub consentHub, Facility facility) throws FacilityAlreadyAssigned {
```

Comment:
Now that we have this method, we probably want to use it in FacilitiesManager in createFacility.